src/main/groovy/org/liquibase/gradle/ProjectInfo.groovy

```diff
@@ -0,0 +1,44 @@
+package org.liquibase.gradle
+
+import org.gradle.api.provider.Provider
+
+class ProjectInfo{
+    private final Map<String, Object> capturedLiquibaseProperties
+    private final String buildDirPath
+    private final Object capturedLogger
+
```

Comment:
💅 

There should be a factory method that produces a `ProjectInfo` from a `Project` (or `LiquibaseTask`)